Main.bb

```diff
@@ -3263,7 +3263,7 @@ Repeat
 		
 		If MsgTimer > 0 Then
 			Local temp% = False
-			If (Not InvOpen%)
+			If (Not (InvOpen Or OtherOpen <> Null)) Then
```

Comment:
I think this is a little difficult to wrap one's head around due to the double negation. Imo something like this would be clearer:

`If (Not InvOpen And OtherOpen = null)`